packages/utilities/psammead-storybook-helpers/src/withServicesKnob.jsx

```diff
@@ -1,21 +1,43 @@
 import React from 'react';
+import path from 'ramda/src/path';
 import { Helmet } from 'react-helmet';
 import { select } from '@storybook/addon-knobs';
 import * as scripts from '@bbc/gel-foundations/scripts';
-import LANGUAGE_VARIANTS from './text-variants';
+import TEXT_EXAMPLES from './text-variants';
+
+const DEFAULT_SERVICE = 'news';
+const SERVICES_LIST = Object.keys(TEXT_EXAMPLES);
+const getVariant = selectedItem => path([selectedItem, 'variant']);
+const getService = selectedItem => path([selectedItem, 'service']);
+const includesService = services => service => services.includes(service);
 
 export default ({
-  defaultService = 'news',
-  services = Object.keys(LANGUAGE_VARIANTS),
+  defaultService = DEFAULT_SERVICE,
+  services = SERVICES_LIST,
 } = {}) => storyFn => {
-  const includesService = service => services.includes(service);
-  const service = select(
+  const selectedItem = select(
```

Comment:
🤔We should give this variable a more descriptive name, `selectedService` perhaps.

Reply:
yeah selectedService is more clear 👍